core/shared/src/main/scala/scalaz/zio/Promise.scala

```diff
@@ -20,40 +20,37 @@ import java.util.concurrent.atomic.AtomicReference
 import scalaz.zio.internal.Executor
 import Promise.internal._
 
-/**
- * A promise represents an asynchronous variable that can be set exactly once,
- * with the ability for an arbitrary number of fibers to suspend (by calling
- * `get`) and automatically resume when the variable is set.
+/** A promise represents an asynchronous variable that can be set exactly once,
+ *  with the ability for an arbitrary number of fibers to suspend (by calling
+ *  `get`) and automatically resume when the variable is set.
  *
- * Promises can be used for building primitive actions whose completions
- * require the coordinated action of multiple fibers, and for building
- * higher-level concurrent or asynchronous structures.
- * {{{
- * for {
+ *  Promises can be used for building primitive actions whose completions
```

Comment:
Extra initial spaces, just 1 space is required between `*` and the first character of the comment.